### DISCUSSION TITLE
BRP has been very helpful

### DISCUSSION THREAD

**Discussion by SteveMaisi** [lassoan/SlicerBoneReconstructionPlanner#62] [General]
Gdday everyone.  I would like to share another surgical fibula planning case.

Surgical planning:
1. Segmental resection
2. Reconstruction with right fibula free flap (anterior pedicle, anastomosis with right neck vessels)

![image](https://user-images.githubusercontent.com/97390689/183421509-5e88803f-9688-4ef7-b2e5-a48edfcfb782.png)
Tumour extension

![image](https://user-images.githubusercontent.com/97390689/183421665-c01c6d55-8df7-4249-a94a-483f316a71f1.png)
Generation of mandibular cutting guide

![image](https://user-images.githubusercontent.com/97390689/183421778-4a16a7b0-45f7-43c2-9dc3-17dfb64a721e.png)
Osteotomy cuts performed

![image](https://user-images.githubusercontent.com/97390689/183426080-30511041-3af2-4b38-9e76-6278642f4f33.png)
Placement of fibula segments

![image](https://user-images.githubusercontent.com/97390689/183422500-4885da27-81cd-441b-bac8-8320092213d7.png)
Final reconstruction

![image](https://user-images.githubusercontent.com/97390689/183422584-9aae435c-ef51-4d11-a97f-c5561595174d.png)
Total fibula length: 360mm

![image](https://user-images.githubusercontent.com/97390689/183422623-f80ddf1d-db72-4b9e-91e1-530334c3ac45.png)
Grey segment to reconstruct the left body of mandible. Lenght: 62mm
Pink segment to reconstruct the anterior mandible. Lenght: 46mm

![image](https://user-images.githubusercontent.com/97390689/183422862-96922425-5cd1-458e-b5d7-2e4b0d20a6a2.png)
Generation of fibula cutting guide

![image](https://user-images.githubusercontent.com/97390689/183422944-55c6649c-c6b1-40dd-b2b8-e7b9b005d773.png)
Final models for 3D printing. 

After using BRP for 4 patients, I think with the correct segmentation It is also possible to use other osseous flaps such as iliac and scapula to reconstruct defects of the mandible/ maxilla.

**Comment by mauigna06** [comment]
Thank you Steve for trusting the software
